### PR TITLE
update slugged-route model unit tests

### LINKED ISSUE #532: Write unit tests for the SluggedRoute model
Write unit tests for the [`SluggedRoute` Model](https://github.com/code-corps/code-corps-ember/blob/develop/app/models/slugged-route.js). There is an [existing unit test file](https://github.com/code-corps/code-corps-ember/blob/develop/tests/unit/models/slugged-route-test.js), but it could use more coverage (computed properties & relationship declarations). 

## References
- Progress on: #98 
- [Model testing guide](https://guides.emberjs.com/v2.8.0/testing/testing-models/)
- [Contributing Guidelines](https://github.com/code-corps/code-corps-ember/blob/develop/.github/CONTRIBUTING.md)
- Join our [Slack](https://codecorps.slack.com)

### PR DESCRIPTION
**Pull request by WenInCode**
# What's in this PR?
Update `slugged-route` unit model tests

## References
- Fixes #532 
- Progress on: #98

### CODE REVIEW

**Comment by JoshSmith** [comment]
These tests are a little confusing in that it makes it look like a `SluggedRoute` always `belongsTo` both a `User` and an `Organization`, which is not true.

Yes, it has these relationships. But there seems to be room for inference here that should be quelled by some clear documentation that it's one or the other, but never both.

**Comment by begedin** [comment]
Same as https://github.com/code-corps/code-corps-ember/pull/547#issuecomment-250958697

It would make sense to add a test that ensures all attributes are accounted for.

**Comment by WenInCode** [comment]
@JoshSmith this is just testing the model logic, and that isn't defined anywhere in the model. I don't think the tests need to reflect that if the logic is not in the model? Or are you suggesting documenting it somewhere?

**Comment by JoshSmith** [comment]
@WenInCode suggesting documenting it, since it looks misleading. If this were my first entrypoint into this model, I wouldn't understand what this is doing .

**Comment by WenInCode** [comment]
Okay cool I can do that

File: tests/unit/models/slugged-route-test.js [hunk 1]
```diff
@@ -10,3 +11,15 @@ test('it exists', function(assert) {
...
+  assert.expect(1);
+
+  let sluggedRoute = this.subject();
+  let attributes = Object.keys(sluggedRoute.toJSON());
+
+  assert.ok(attributes.includes('slug'), 'should have the slug attribute');
+});
+
+testForBelongsTo('slugged-route', 'user');
+testForBelongsTo('slugged-route', 'organization');
```
Comment (JoshSmith) [inline tests/unit/models/slugged-route-test.js]: Alpha order.

File: tests/unit/models/slugged-route-test.js [hunk 2]
```diff
@@ -21,5 +21,7 @@ test('should have all of its attributes', function(assert) {
   assert.ok(attributes.includes('slug'), 'should have the slug attribute');
 });
 
-testForBelongsTo('slugged-route', 'user');
+// slugged-route may belong to either a user or organization, but not both
+// at the same time.
```
Comment (JoshSmith) [inline tests/unit/models/slugged-route-test.js]: 👍

**Comment by JoshSmith** [comment]
Rebase, squash, merge.

**Pull request merged**